lib/src/map/camera/camera.dart

```diff
@@ -204,6 +219,21 @@ class MapCamera {
     return Point<double>(width, height);
   }
 
+  /// Jumps camera to opposite side of the world to enable seamless scrolling
+  /// between 180 and -180 longitude.
+  LatLng adjustPositionForSeamlessScrolling(LatLng position) {
+    double adjustedLongitude = position.longitude;
+    if (seamlessPanning != null && seamlessPanning!) {
+      if (adjustedLongitude >= 180.0) {
+        adjustedLongitude = adjustedLongitude - 360.0;
+      } else if (adjustedLongitude <= -180.0) {
+        adjustedLongitude = adjustedLongitude + 360.0;
+      }
```

Comment:
Some side-effects here: if it's 180 you want -180, if it's -180 you want 180. Would be better if you decide which one you prefer.